sdcm/cluster.py

```diff
@@ -1966,16 +1966,21 @@ def install_manager_agent(self, package_path=None):
                 self.parent_cluster.params.get("scylla_mgmt_repo"))
             package_name = 'scylla-manager-agent'
         package_mgr = "yum" if self.distro.is_rhel_like else "apt-get"
+
+        @retrying(n=5, sleep_time=2, allowed_exceptions=UnexpectedExit)
+        def install_package():
+            self.remoter.run(f'{package_mgr} install -y {package_name}')
+
+        install_package()
```

Comment:
May be better to make this internal function "install package" as class "node" method?

Reply:
use self.remoter.sudo

Reply:
The idea of this PR is to add something fast to fix these failures, as they are preventing us to get results.
I agree with you that we should have a more generic mechanism for "install package", but it will be a much wider change.
I'm opening a task in Trello for it (https://trello.com/c/ceWDHLJ7/3268-add-a-generic-installpackage)